operations/cancellationpolicies.md

```diff
@@ -102,6 +100,176 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+
+For existing reservations, supply the reservation ids. This is because the cancellation policies are locked per reservation when the reservation is created. That means the reservaton's cancellation policies are not changed when cancellation policy is created, updated or deleted.
+
+If the response do not contain any cancellation policy for any given reservation draft parameters or reservation id, no cancellation policies apply.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/cancellationPolicies/getAll/2024-09-02`
+
+```javascript
+{
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "Client": "Sample Client 1.0.0",
+  "ReservationDraftParameters": [
+    {
+      "ReservationFirstTimeUnitStartUtc": "2024-06-01T00:00", 
+      "ReservationLastTimeUnitStartUtc": "2024-06-05T00:00",    
+      "RateIds": [
+        "c5bb9d3e-849d-4119-a6bc-ac87a2b20810",
+        "8db45d0a-9bec-4550-ad67-ed0265bcacce"
+      ]
+    },
+    {
+      "ReservationFirstTimeUnitStartUtc": "2024-06-21T00:00", 
+      "ReservationLastTimeUnitStartUtc": "2024-06-28T00:00",    
+      "RateIds": [
+        "c5bb9d3e-849d-4119-a6bc-ac87a2b20810",
+        "8db45d0a-9bec-4550-ad67-ed0265bcacce"
+      ]
+    }
+  ],
+  "ReservationIds": [
+    "bcfce146-ea7b-4601-baed-5f0e09e0ba2d",
+    "38f76277-9c66-4e9f-850c-db708f489a1d"
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ReservationDraftParameters` | array of [Reservation Draft Parameteres](#reservation-draft-parameters) | optional, max 100 items | Reservation draft parameters used for reservations that has not been confirmed yet. |
+| `ReservationIds` | array of string | optional, max 100 items | Unique identifiers of reservations. |
+
+
+#### Reservation Draft Parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ReservationFirstTimeUnitStartUtc` | string | required | Start of the time reservation interval, expressed as the timestamp for the start of the last [time unit](services.md#time-unit), in UTC timezone ISO 8601 format. |
+| `ReservationLastTimeUnitStartUtc` | string | required | End of the time reservation interval, expressed as the timestamp for the start of the last [time unit](services.md#time-unit), in UTC timezone ISO 8601 format. |
+| `RateIds` | array of string | required, max 100 items | Unique identifiers of requested rates. |
+
+### Response
+
+```javascript
+{
+  "CancellationPoliciesByReservationDraft": [
```

Comment:
CancellationPoliciesByReservationDraft - we are exposing internal implementation here with this name, usually not a good idea